ui/src/main/webapp/home/index.js

```diff
@@ -113,31 +66,36 @@ const LdapTileView = (props) => {
     baseUserDn,
     onDeleteConfig,
     ldapUseCase,
-    attributeMappings
+    attributeMappings,
+    submitting
   } = props
 
   return (
-    <Paper className={styles.config}>
-      <div className={styles.tileTitle}>{ldapUseCase === 'authentication' ? 'LDAP Authentication Source' : 'LDAP Attribute Store'}</div>
-      <ConfigField fieldName='Hostname' value={hostName} />
-      <ConfigField fieldName='Port' value={port} />
-      <ConfigField fieldName='Encryption Method' value={encryptionMethod} />
-      <ConfigField fieldName='Bind User' value={bindUser} />
-      <ConfigField fieldName='Bind User Password' value='******' />
-      <ConfigField fieldName='UserName Attribute' value={userNameAttribute} />
-      <ConfigField fieldName='Base Group DN' value={baseGroupDn} />
-      <ConfigField fieldName='Base User DN' value={baseUserDn} />
-      { ldapUseCase !== 'authentication'
-        ? (
-          <MapDisplay label='Attribute Mappings'
-            mapping={attributeMappings} />
-      ) : null }
-      <RaisedButton style={{marginTop: 20}} label='Delete' secondary onClick={onDeleteConfig} />
+    <Paper className={styles.outerSpinner}>
+      <Spinner submitting={submitting}>
+        <div className={styles.innerSpinner}>
```

Comment:
We become very brittle if we leak this much information up. Plus, leaky.

I like the idea of adding some component to each wizard that can be plugged into the homepage; I'd like to figure out some way of whiteboarding the knowledge of what components are available for the homepage to then display.

If we can't, I have no idea how this becomes of general value because any developer who wants to add a wizard for some new functionality they've developed has to modify this code. So Joe Blow at the World Wide Widget company can't just deploy his feature(s) and his configuration wizard, he has to replace our configuration home page feature with his own flavor of it.